style.css

```diff
@@ -0,0 +1,118 @@
+body {

```

Comment:
for all file. You need to use properties according to the rule about the order of css properties. Firstly about positioning, then about box model etc.